### PR TITLE
added equation for calculating credits

### PR DESCRIPTION
**Pull request by ndoubleg**

### CODE REVIEW

File: Test.C [hunk 1]
```diff
@@ -150,14 +150,35 @@ int main(){
...
-        }  //current encrypt
+            printf("%s\n", encryptionPointer->encryption->name); //current encrypt
+            encryptionPointer=encryptionPointer->next; //pointer
+        } //end while loop
+		
         //iterate and print through current encryption
-
-        //receive credits NATHAN
-        //use equation to figure out how many credit user receives
+    }
```
Comment (eholland2000) [inline Test.C]:
```suggestion

```

Comment (eholland2000) [reply Test.C]: I believe when merging in the other pull request, I may have accidentally added this

File: Test.C [hunk 2]
```diff
@@ -150,14 +150,35 @@ int main(){
...
+        
+    //receive credits
+		int sum;
+		int newCredits;
```
Comment (eholland2000) [inline Test.C]:
```suggestion
		int sum = 0;
		int newCredits = 0;
```
Clears out garbage data

File: Test.C [hunk 3]
```diff
@@ -150,14 +150,35 @@ int main(){
...
+		printf("Original credits: %d\n", player.currentCredits); //prints out the current credits the player has
+
+    //use equation to figure out how many credit user receives
+		newCredits = player.currentCredits * Encryption.effectiveness * Equipment.messageAmount;
```
Comment (eholland2000) [inline Test.C]:
```suggestion

```
Seems like this was left as pseudocode but you have it implemented below so we don't need it

File: Test.C [hunk 4]
```diff
@@ -150,14 +150,35 @@ int main(){
...
+		
+		for (int i = 0; i < equipment; i++){
```
Comment (eholland2000) [inline Test.C]:
```suggestion
		for (int i = 0; i < NUM_EQUIPMENT; i++){
```

File: Test.C [hunk 5]
```diff
@@ -150,14 +150,35 @@ int main(){
...
+			sum += equipmentStock[i] * equipmentInventory[i]; //sum message amount of equipment(s)
```
Comment (eholland2000) [inline Test.C]:
```suggestion
			sum += equipmentStock[i].messageAmount * player.equipmentInventory[i]; //sum message amount of equipment(s)
```

**Review by eholland2000 — CHANGES_REQUESTED** [review]
Just fixed some things so it would compile. We have to change how we implemented the credit calculation, but in order to do that, it requires modifying the equipment data structure so I will do that in another code review. The rest looks good.

File: Test.C (hunk 3 above)
Comment (ndoubleg) [reply Test.C]: yeah, this was what we talked about before we did all that stuff on the zoom call

**Comment by ndoubleg** [comment]
if everything looks good to you go ahead and merge it

**Review by eholland2000 — APPROVED** [review]

**Pull request merged**